modular_skyrat/code/_HELPERS/roundend.dm

```diff
@@ -0,0 +1,23 @@
+/datum/controller/subsystem/ticker/declare_completion()
+	..()
+	for(var/M in GLOB.player_list)
+		var/mob/m = M
+		if (!m.client)
+			continue
+		m.client.verbs += /client/proc/eorg_teleport
+		if (m.client && m.client.prefs && m.client.prefs.eorg_teleport)
+			m.forceMove(pick(GLOB.eorg_teleport).loc)
+			to_chat(m, "<BR><span class='narsiesmall'>You have chosen not to EORG. Do not commit EORG or you will be banned!</span>")
+			to_chat(m, "<BR><BR><span class='notice'>You have been successfully recovered from SS13 and are on your way to the nearest transit interchange. There's some time before the next shuttle home comes though, time to relax.</span>")
+			to_chat(m, "<span class='greentext'>You've made it.</span>")
+
+/client/proc/eorg_teleport()
+	set name = "Go Home"
+	set category = "OOC"
+	set desc = "Teleport to a no EORG area."
+	var/mob/living/carbon/human/H = mob
+	H.revive(full_heal=TRUE,admin_revive=TRUE)
+	H.unbuckle_mob()
+	mob.forceMove(pick(GLOB.eorg_teleport).loc)
```

Comment:
since you are already using H which is mob, why not use H.forceMove rather than mob.forceMove

Reply:
Thanks, sorted that.